modules/cas/lib/Auth/Source/CAS.php

```diff
@@ -78,6 +78,29 @@ public function __construct($info, $config) {
 		}else{
 			throw new Exception("cas login URL not specified");
 		}
+
+
+		// Check for the required ldap configuration 
+		if(!isset($this->_ldapConfig['servers'])) {
+			throw new Exception("ldap server not specified");
+		}
+
+		// Check for ldap options in cas configuration if not configured set to default
```

Comment:
We are setting defaults here that are also specified in the SimpleSAML_Auth_LDAP constructor as well. This seems suboptimal to repeat in different places of the code.

But I don't have a clear-cut solution to that in the context of just the CAS module, so it might be acceptable here. Maybe SimpleSAML_Auth_LDAP's constructor should be changed so that if you pass in `null` for any of the optional params, you get the default?